Dockerfiles/gadget-core.Dockerfile

```diff
@@ -42,26 +42,18 @@ RUN cd /gadget/gadget-container && \
 # Main gadget image
 FROM ${BASE_IMAGE}
 
-# install runtime dependencies  according to the package manager
-# available on the base image
 RUN set -ex; \
-	if command -v tdnf; then \
-		tdnf install -y libseccomp wget util-linux; \
-	elif command -v yum; then \
-		yum install -y libseccomp wget util-linux; \
-	elif command -v apt-get; then \
-		apt-get update && \
-		apt-get install -y seccomp wget util-linux; \
-	elif command -v apk; then \
-		apk add gcompat libseccomp wget util-linux; \
-	fi && \
 	(rmdir /usr/src || true) && ln -sf /host/usr/src /usr/src && \
 	rm -f /etc/localtime && ln -sf /host/etc/localtime /etc/localtime
 
 COPY gadget-container/entrypoint.sh gadget-container/cleanup.sh /
 
 COPY --from=builder /gadget/gadget-container/bin/gadgettracermanager /bin/
 
+# We need to copy libseccomp.so and libdl.so from builder image as
+# gadgettracermanager relies on them.
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libseccomp.so.2 /lib/x86_64-linux-gnu/libdl.so.2 /lib
+
```

Comment:
Is this safe? Can we always use this regardless of the base image we use to build our container?

Reply:
I think it is not safe in general: `*.so` files can be compiled with different options across-distros so they might not have the same dependencies.

@eiffel-fl Did you have a specific issue with DNS in the gadget-core image? With wget in entrypoint.sh downloading from btfhub?